enterprise/app/models/applied_sla.rb

```diff
@@ -21,5 +21,7 @@ class AppliedSla < ApplicationRecord
   belongs_to :sla_policy
   belongs_to :conversation
 
+  validates :account_id, uniqueness: { scope: %i[sla_policy_id conversation_id] }
```

Comment:
We should add a migration to the back this with unique index

Reply:
Noted